apps/admin-ui/src/spa/unit/[id]/ReservationUnitCard.tsx

```diff
@@ -113,8 +114,7 @@ export function ReservationUnitCard({
 
   const hasPurposes = (reservationUnit?.purposes?.length || 0) > 0;
 
-  // TODO use urlBuilder
-  const link = `/unit/${unitId}/reservationUnit/edit/${reservationUnit.pk}`;
+  const link = getReservationUnitUrl(reservationUnit.pk, unitId);
```

Comment:
`?` is it edit or the reservationUnit page? what tells the difference in url builder?

Reply:
yeah it's correct, naming is maybe questionable